### PR TITLE
Frontend Challenge v2 (2020)

### PR DESCRIPTION
**Pull request by ncounter**
New challenge description for the [new open position](https://jobs.suse.com/us/en/job/7012993/Senior-Frontend-Engineer-Flexible-Location)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,27 +29,37 @@ You can choose your own path among the following levels:
 
 **TASKS**
 + *All of the base requirements*
-+ Create and implement (using HTML, CSS and JS) a UI for listing the [short list of systems](https://github.com/SUSE/frontend-challenge/blob/master/systems-short-list.json) systems with a basic design
++ Use a JavaScript framework ([ReactJS](https://reactjs.org/) is highly recommended)
```
Comment (cavalheiro) [inline README.md]: I would say the use of React is mandatory

File: README.md [hunk 2]
```diff
@@ -31,27 +29,37 @@ You can choose your own path among the following levels:
...
 ## Intermediate Level
 
 **TASKS**
 + *All of the base and entry requirements*
-+ Use a JavaScript framework (ReactJS is preferred)
-+ Use a CSS Framework (Bootstrap is preferred)
-+ Use LESS for custom CSS
++ Use [Yarn](https://yarnpkg.com/) to set up the project
 + Cross browser support (or graceful degradation)
-+ Responsive design and implementation
```
Comment (cavalheiro) [inline README.md:44]: Why are we considering responsive design only on the advanced level?

**Review by cavalheiro — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ncounter) [reply README.md]: Agreed, it makes sense for the purpose of the challenge :+1:

File: README.md (hunk 2 above)
Comment (ncounter) [reply README.md:44]: Well, on one hand responsive design is not a big deal form the technical POV, but it is also something that can annoy the implementation without focusing on other tasks. We could move to the `intermediate` maybe? Basically it is just more in line with our current situation: we don't have a responsive product interface. Fine to move to intermediate?

Comment (ncounter) [reply README.md:44]: Moved to `intermediate` for now.

File: README.md (hunk 1 above)
Comment (ncounter) [reply README.md]: Made ReactJS mandatory.

File: README.md [hunk 3]
```diff
@@ -31,27 +29,37 @@ You can choose your own path among the following levels:
...
-+ Host the website on the service of your choice (We enjoy cloud hosting like Azure or AWS, but we would also settle for a *.github.io address)
++ Include some graph-like visualization of the data
++ Host the website on the service of your choice (We enjoy cloud hosting like Azure or AWS, or a *.github.io address)
++ More statistics reports:
+  + Branches
+  + Forks
+  + Contributors
+  + ..
 
 ## Bonus
```
Comment (cavalheiro) [inline README.md:64]: Could we maybe remove this section is there is nothing in it?

**Review by cavalheiro — APPROVED** [review]

**Pull request merged**